src/robot/rebot.py

```diff
@@ -201,6 +201,13 @@
                           match (starting from 1).
                           Examples: --tagstatlink mytag:http://my.domain:Title
                           --tagstatlink "bug-*:http://url/id=%1:Issue Tracker"
+    --expandkeywords name:<pattern>|tag:<pattern> *
+                          Matching keywords will be initially expanded in the 
+                          log file.
+                          For pattern matching rules see --removekeywords
+                          option.
+                          Example: --expandkeywords name:CapturePageScreenshot
+                          This is new in RF 3.2.
```

Comment:
This is formatted quite a bit differently than other help texts. Starting the doc after the line with the option itself is fine with this long option, but doc should continue on same line after `log file.`.

Reply:
you a re right. this is wanted: i followed the style of removekeywords and flattenkeywords to be consistent inside of rebot.py. i raised a more general *.rst vs. *.py formatting issue to harmonize in more general => https://github.com/robotframework/robotframework/issues/3378